sequencer/src/persistence/sql.rs

```diff
@@ -47,6 +53,20 @@ pub struct Options {
     /// Use TLS for an encrypted connection to the database.
     #[clap(long, env = "ESPRESSO_SEQUENCER_POSTGRES_USE_TLS")]
     pub use_tls: bool,
+
+    /// This will enable the pruner and set the default pruning parameters unless provided.
+    /// Default parameters:
+    /// - pruning_threshold: 100GB
+    /// - minimum_retention: 1 day
+    /// - target_retention: 30 days
+    /// - batch_size: 1000
+    /// - max_usage: 80%
```

Comment:
Our default parameters should match the [recommended hardware](https://docs.espressosys.com/sequencer/guides/running-a-sequencer-node#hardware-requirements), based on [this policy](https://github.com/EspressoSystems/hotshot-query-service/issues/376#issuecomment-1917640481):
* pruning threshold: 3 TB
* minimum retention: 1 day
* target retention: 7 days

Reply:
I don't have a strong opinion on interval. We can run it for a bit in staging and see what works. But 1 hour seems like a reasonable starting place.